src/routes.py

```diff
@@ -72,3 +72,17 @@ def logout() -> Response:
         session.delete(session_to_delete)
         session.commit()
     return redirect(url_for("routes.login"))
+
+
+@bp.route("/topics")
+def topics() -> str | Response:
+    """Handle topics page."""
+    session_id = request.cookies.get("session_id")
+    user_session = session.query(UserSession).filter_by(session_id=session_id).first()
+
+    current_user = None
+    if user_session is not None:
+        current_user = session.query(User).filter_by(id=user_session.user_id).one()
+        topic_list = session.query(Topic).join(User).order_by(Topic.created_at.desc()).all()
```

Comment:
We will not need this `.join(User)` clause after we add relationship to `Topic` model. So just mention it here to not forgot.

Reply:
Yes, relationship field works like a join. I'll try to explain. This task could be done in two ways:
1. Using explicit `join`
```python
# somewhere in routes.py

topic_list = session.query(
    Topic.title,
    Topic.description,
    Topic.created_at,
    User.username
).join(User).order_by(Topic.created_at.desc()).all()
```
```html
<!-- somewhere in topics.html -->

{% for topic in topic_list %}
    <ul>
        <li><b>{{ topic.title }}</b></li>
        <li>{{ topic.description }}</li>
        <li>
            {{ topic.username }} said
        </li>
        <li>at {{ topic.created_at }}</li>
    </ul>
{% endfor %}
```